src/views/Analysis.vue

```diff
@@ -28,14 +28,20 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
           md="4"
           class="pr-md-2 mb-2 mb-md-0"
         >
-          <v-text-field
-            id="c-analysis-filter-task-name"
-            clearable
-            placeholder="Filter by task name"
-            v-model.trim="tasksFilter.name"
-            ref="filterNameInput"
-            :disabled="chartType === 'timeSeries'"
-          />
+          <v-tooltip>
+            <template #activator="{ props }">
+              <v-text-field
+                id="c-analysis-filter-task-name"
+                clearable
+                placeholder="Filter by task name"
+                v-model.trim="tasksFilter.name"
+                ref="filterNameInput"
+                :disabled="chartType === 'timeSeries'"
+                v-bind="props"
+              />
```

Comment:
We should be able to make the space separated bit a little more intuitive.

Check out the Vuetify Combobox component, it has a [multiple option](
https://vuetifyjs.com/en/components/combobox/#multiple-combobox) that allows multiple values to be specified (note you don't *have* to pre-populate suggestions to a combox). It can place each item in a "chip" and allow each item [to be individually clearable](https://vuetifyjs.com/en/components/combobox/#advanced-custom-options). It should also take care of the `/\s/` stuff.

Reply:
Done